### PR TITLE
Experimental dynamic trees, vmaps, mmaps from TC (RE) for AzerothCore

### PR DESCRIPTION
**Pull request by kvipka**
In Sunwell by default :
- no correct texture of vmaps under player

How it should be:

Be carefuly: PR not completed !

-  
-  
-  

**Target branch(es):** 1.x/2.x etc.

**Issues addressed:** Closes #

**Tests performed:** (Does it build, tested in-game, etc)

**Known issues and TODO list:**

- [ ] 
- [ ] 

**NOTE** You no longer need to squash your commits, on merge we will squash it for you. (GitHub added a new feature)

### CODE REVIEW

File: modules/acore/extractors/vmap4_extractor/adtfile.cpp [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
```
Comment (ShinDarth) [inline modules/acore/extractors/vmap4_extractor/adtfile.cpp]: please revert this kind of change of the file headers

**Comment by Inifield** [comment]
"src/cmake/FindBoost.cmake" AC does not use Boost

**Comment by sucofog** [comment]
ACE switch to boost @Inifield

**Comment by Yehonal** [comment]
@sucofog and I worked on another solution for getHeight that doesn't require boost for now https://github.com/azerothcore/azerothcore-wotlk/pull/705

we will include fix for liquid too in future, but we can easily use c++17 function instead of boost since new c++ version includes them.

**Comment by Yehonal** [comment]
it's a mess, i've solved conflicts, fixed cmake and still have tons of errors....is it a never compiled cherry pick?

please fix it before

closed

**Pull request closed without merging**